Ginger/Ginger/Actions/ApiActionsConversion/ApiConversionConfigurationWzardPage.xaml

```diff
@@ -0,0 +1,33 @@
+<Page x:Class="Ginger.Actions.ApiActionsConversion.ApiConversionConfigurationWzardPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+      xmlns:Ginger="clr-namespace:Ginger"
+      xmlns:GingerCore="clr-namespace:GingerCore;assembly=GingerCore"
+      mc:Ignorable="d" 
+      d:DesignHeight="450" d:DesignWidth="700"
+      Title="ApiConversionConfigurationWzardPage">
+
+    <Grid Background="{StaticResource $BackgroundColor_LightGray}" x:Name="xGrdPane">        
+        <Grid.RowDefinitions>
+            <RowDefinition Height="40" />
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="450" />
+            <ColumnDefinition Width="*" />
+        </Grid.ColumnDefinitions>
+
+        <CheckBox x:Name="xChkParameterizeRequestBody"  Grid.Column="0" Height="20" Grid.Row="0" IsChecked="{Binding Path=ParameterizeRequestBody, Mode=TwoWay}" HorizontalAlignment="Left"  Margin="10,20,0,0" >
+            <CheckBox.Content>
+                <GingerCore:ucTextDicResource Text="Parameterize the Request body"/>
```

Comment:
1- for both check boxes make the text with low letters, no need capital except first letter
2- add tooltip for both explaining more in details what each means